bubble_sort.rb

```diff
@@ -0,0 +1,40 @@
+unsorted_list = [6, 2, 8, 1, 7, 3, 4]
```

Comment:
- [ ] Please remove the debugging code before submiting for review. You should submit only the methods. (1)